### ISSUE TITLE
No new articles / Feeds

### ISSUE THREAD

**Issue by derunterstrich** [FreshRSS/FreshRSS#2629]
Whether via the API, or via the web: There are no new articles coming in. I get a message via the web:

There is no feed to update...

I have 56 feeds in 6 categories. If I click on the icon next to the feed name and then select Update, the new articles will be imported. 

What am I doing wrong, or where is the error?

I already updated to the new version 1.15

**Comment by weado** [comment]
Hello, it similar for me : 
When I go to the webpage, the home page show me there aren't new articles, and when I reload my page, the new articles appear.
It's the same when I click on the "read all articles", I have to click twice to make the action correctly.

I try with and without reverse proxy.

**Comment by Alkarex** [comment]
@derunterstrich Do you have a cron job running to update your feeds?

@weado I do not think your case is the same. It looks more like a cache problem. If you like, you are welcome to send me a link and temporary account to your instance so I can try

**Comment by fabiosirna** [comment]
It happens to me as well.
I noticed that when my connection is down, FreshRSS is not able to update the feeds, marks them in red and the categories in red with an escalmation dot triangle.
If I try to refresh the feeds from the top right icons (circular arrows) I got the same message in this thread.
The only way to get your feeds is to manually update EACH feed:
![image](https://user-images.githubusercontent.com/2249234/161414837-910ba992-9979-4718-9155-a796c978ac4d.png)

Maybe this is a bug or the opportunity to add a 'update all feeds' button to force all the updates.

**Comment by math-GH** [comment]
It is the mindset of FreshRSS, that you cannot poll the feeds so often (to keep the resource usage of FreshRSS and feed server low).

In the configs in section "Archiving" you can edit the time for all feeds and in the subscriptions management individually for each feed.

**Comment by fabiosirna** [comment]
@math-GH thanks for pointing this out. My default is 1h but when the internet connection is down, FreshRSS marks the feeds in red, and stop pulling the feeds. When the internet connection is up, even after 1h they are still red. The only way for me to get them back working is to manually update. What I'm asking for is to have a way to force manually all the feeds so that I don't have to do it for every feed. It will used only in case of issues.

**Comment by karimone** [comment]
> [@math-GH](https://github.com/math-GH) thanks for pointing this out. My default is 1h but when the internet connection is down, FreshRSS marks the feeds in red, and stop pulling the feeds. When the internet connection is up, even after 1h they are still red. The only way for me to get them back working is to manually update. What I'm asking for is to have a way to force manually all the feeds so that I don't have to do it for every feed. It will used only in case of issues.

I have the same problem. Is there a way to batch force the reload of every feed?

**Comment by Alkarex** [comment]
> I have the same problem. Is there a way to batch force the reload of every feed?

Yes, just click on the refresh icon, or do nothing and let your cron job refresh the feeds according to their respective policies. Note however, as answered higher up, that the refresh cannot be made shortly after the feed was already refreshed, but (error or not) need to wait according to the feeds respective refresh policies. With default settings, that means one hour later.

A better fix would be to distinguish the type of error, and in case of local network error to make it not count as a refresh. PR welcome.
